source/common/network/utility.cc

```diff
@@ -342,5 +342,30 @@ bool Utility::portInRangeList(const Address::Instance& address, const std::list<
   return false;
 }
 
+absl::uint128 Utility::Ip6ntohl(const absl::uint128& address) { return flipOrder(address); }
+
+absl::uint128 Utility::Ip6htonl(const absl::uint128& address) { return flipOrder(address); }
+
+std::array<uint8_t, 16> Utility::getArrayRepresentation(const absl::uint128& address) {
```

Comment:
I think this method is not needed. memcpy should work where this is used.

Reply:
Updated all call paths to use memcpy.